README.md

```diff
@@ -11,6 +11,9 @@ Tested with kubernetes v1.16.2 and v1.17.0
 
 Tested with cert-manager v0.13.0
 
+[helm](https://helm.sh/) is not strictly a prerequisite, as there are other
+options for managing charts, but we use it in our examples.
```

Comment:
Is this true? There are definitely other templating tools, but I think Charts are a helm concept?

Reply:
yeah good point